autorag/data/corpus/llama_index.py

```diff
@@ -26,11 +27,16 @@ def llama_documents_to_parquet(llama_documents: List[Document],
         Default is False.
     :return: Corpus data as pd.DataFrame
     """
-    doc_lst = list(map(lambda doc: {
-        'doc_id': str(uuid.uuid4()),
-        'contents': doc.text,
-        'metadata': add_essential_metadata(doc.metadata)
-    }, llama_documents))
+    doc_ids = [str(uuid.uuid4()) for _ in llama_documents]
+    doc_lst = [
+        {
+            'doc_id': doc_id,
+            'contents': doc.text,
+            'metadata': add_essential_metadata(doc.metadata, prev_id, next_id)
+        }
+        for doc, doc_id, prev_id, next_id in zip(llama_documents, doc_ids, [None] + doc_ids[:-1], doc_ids[1:] + [None])
```

Comment:
I think we cannot guarantee any prev-next relationship between each Llama Documents, right?
Just set prev_id and next_id to None, or just don't insert them since it will process at`cast_corpus_data`